File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
    int maxwealth = 0;
        int currentwealth = 0;
        for(int i = 0 ; i < accounts.length ; i++){
            currentwealth = 0;
            for(int j = 0; j < accounts[i].length;j++){
                currentwealth += accounts[i][j];
            }
            if(currentwealth > maxwealth){
                maxwealth = currentwealth;
            }
           
        }
         return maxwealth;
}
}
    

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
     let maxwealth = 0;
        for(let i = 0 ; i < accounts.length ; i++){
            let currentwealth = 0;
            for(let j = 0 ; j < accounts[i].length ; j++){
                currentwealth += accounts[i][j];
            }
           if(currentwealth > maxwealth){
               maxwealth = currentwealth;
           }
        }
    return maxwealth;
};